assets/src/components/settingsPage.tsx

```diff
@@ -30,13 +27,10 @@ const SettingsPage = (): ReactElement<HTMLDivElement> => {
     <div className={`l-page c-settings-page ${mobileMenuClass}`}>
       <Notifications />
       <div className="c-page__container">
-        <h1 className="c-page__title">Settings</h1>
-
```

Comment:
The Figma doc is a unclear on this, but I'd check with design that they wanted the `Settings` heading removed

Reply:
Good call! I'll doublecheck